wasm2c/examples/rot13/main.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
```

Comment:
Top level comment describing that this is doing ?     Something like `entry point for rot13 program which loads and runs the wasm rot13 program as compiled into C by wasm2c`?

Reply:
Done.